cmd/skopeo/sync.go

```diff
@@ -723,6 +726,15 @@ func (opts *syncOptions) run(args []string, stdout io.Writer) (retErr error) {
 		logrus.Warn("Running in dry-run mode")
 	}
 
+	var digestFile *os.File
+	if opts.digestFile != "" && !opts.dryRun {
+		digestFile, err = os.OpenFile(opts.digestFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
```

Comment:
Why `O_APPEND` here? It’s not the typical behavior when providing a destination path to a program.

It’s not _completely unheard of_ but it seems unusual to me.